### PR TITLE
Create json web token for login route

### PR DESCRIPTION
**Pull request by berlinum**
Install JSON web token from npm
Create a token key and store it in .env

#11

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,2 +1,3 @@
 PORT=8080
+jwtSecret=maxim_bricks_2020
 DB_URL=mongodb+srv://berlinum:XpYXHlabIFVtD3Qq@bricks-vv3is.mongodb.net/test?retryWrites=true&w=majority
```
Comment (lmachens) [inline .env:3]: This file should not be in git.
Now I know your password ;).

File: .env [hunk 2]
```diff
@@ -1,2 +1,3 @@
 PORT=8080
+jwtSecret=maxim_bricks_2020
```
Comment (lmachens) [inline .env:2]: Environments are usually in UPPERCASE.
```suggestion
JWT_SECRET=maxim_bricks_2020
```

File: .env (hunk 1 above)
Comment (berlinum) [reply .env:3]: 😣

File: .env (hunk 2 above)
Comment (berlinum) [reply .env:2]: Fixed!

**Pull request merged**